apps/files_external/lib/AppInfo/ConfigLexicon.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Files_External\AppInfo;
```

Comment:
```suggestion
namespace OCA\Files_External\Config;
```

I suggest we put this in `Config` namespace (dir).

Reply:
I agree, the AppInfo namespace should not be poluted.

Reply:
I will move it into `lib/`